apps/easy-messe/components/layout/Header/Header.tsx

```diff
@@ -69,7 +77,7 @@ export default function Header() {
                     <IconButton sx={{
                         display: { laptop: 'none', mobile: 'block' }
                     }}
-                        onClick={() => setIsSideBarOpen(true)}
+                        onClick={() => openSideBar()}
```

Comment:
Already mention in the last review. Don't use arrow function. Just call the funciton.